### DISCUSSION TITLE
Migration from Trinity to firefly

### DISCUSSION THREAD

**Discussion by KK887788** [iotaledger/firefly#7670] [Help & Support]
Hey there,

after a year of inactivity I tried to check on my Miotas. I downloaded Firefly, it asked me for my S´Wordphrase. I only have a Seed from Trinity though. 

After I downloaded an older Version of Firefly, it asked me for the Seed but after I enter it, Firefly says that a problem occured while restoring my wallet. 

Did anyone experience the same and what can I do now?

**Comment by begonaalvarezd** [comment]
The current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again
We will provide a migration tool to resume legacy migrations, but its not finished yet.
Will close the issue since this will not be included in Firefly
